intelmq/bots/experts/rdap/expert.py

```diff
@@ -99,11 +99,19 @@ def process(self):
                             raise NotImplementedError("Authentication type %r (configured for service %r) is not implemented" % (service['auth'], domain_suffix))
 
                     resp = self.__session.get("{0}domain/{1}".format(service['url'], url))
+
+                    if not 200 <= resp.status_code <= 299:
```

Comment:
This line looks rather obfuscated. `if resp.status_code < 200 and resp.status_code > 299` seems more readable to me.

Reply:
Why not just `resp.ok`?

Reply:
Or just use the `resp.raise_for_status()`